symplyphysics/laws/gravity/keplers_constant_via_attracting_body_mass.py

```diff
@@ -0,0 +1,54 @@
+"""
+Kepler's constant via attracting body mass
+==========================================
+
+Kepler's constant is a physical quantity that only depends on the gravitational constant and the mass of
+the orbited body, such as the Sun. It is constant in the sense that all planets that orbit the same body
+approximately have the same value of the Kepler's constant.
+"""
+
+from sympy import Eq, pi
+from sympy.physics.units import gravitational_constant
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+    symbols,
+    clone_symbol,
+)
+
+keplers_constant = Symbol("keplers_constant", units.length**3 / units.time**2)
+r"""
+Symbol:
+    K
+
+Latex:
+    :math:`\mathfrak{K}`
+"""
+
+attracting_body_mass = clone_symbol(symbols.basic.mass, "attracting_body_mass")
+"""
+The mass of the attracting body.
```

Comment:
Write `mass` as link to mass symbol, like here https://github.com/blackyblack/symplyphysics/blob/8cb741a9c316c0f1dca0c2509a20244f3d2edde1/symplyphysics/laws/kinematic/vector/centrifugal_acceleration_via_centripetal_acceleration.py#L29